files/scripts/bgp.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+SERVICE="bgp"
+DEBUGLOG="/tmp/bgp_debug.log"
+
+function debug()
+{
+    /usr/bin/logger $1
+    /bin/echo `date` "- $1" >> ${DEBUGLOG}
+}
+
+function check_warm_boot()
+{
+    SYSTEM_WARM_START=`/usr/bin/redis-cli -n 6 hget "WARM_RESTART_ENABLE_TABLE|system" enable`
+    SERVICE_WARM_START=`/usr/bin/redis-cli -n 6 hget "WARM_RESTART_ENABLE_TABLE|${SERVICE}" enable`
+    if [[ x"$SYSTEM_WARM_START" == x"true" ]] || [[ x"$SERVICE_WARM_START" == x"true" ]]; then
+        WARM_BOOT="true"
+    else
+        WARM_BOOT="false"
+    fi
+}
+
+function validate_restore_count()
+{
+    if [[ x"$WARM_BOOT" == x"true" ]]; then
+        RESTORE_COUNT=`/usr/bin/redis-cli -n 6 hget "WARM_RESTART_TABLE|bgp" restore_count`
```

Comment:
use sonic-db-cli instead

Reply:
Changed  "/usr/bin/redis-cli -n 6" to "$SONIC_DB_CLI STATE_DB"